Android/app/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
         applicationId "live.dittolive.chat"
         minSdk 29
         targetSdk 34
-        versionCode 33
+        versionCode 34
         versionName "1.0.1"
```

Comment:
Sorry I am just now seeing this, but we should probably also bump version name as well